Gems/Atom/Asset/Shader/Config/Platform/Linux/shader_build_options.json

```diff
@@ -7,6 +7,7 @@
         ],
         "dxc": ["-spirv"
               , "-fvk-use-dx-layout" // Use the same memory layout as DX12, otherwise some offset of constant may get wrong.
+              , "-fspv-target-env=vulkan1.1"
```

Comment:
Again.  I would be hesitant to add this without testing Linux as this change will impact all Linux shaders.